src/main/java/ch/njol/skript/expressions/ExprAttacked.java

```diff
@@ -65,8 +70,8 @@
 
 	@Override
 	public boolean init(final Expression<?>[] vars, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
```

Comment:
Currently, if `SUPPORT_PROJECTILE_HIT` is false, `victim` is still usable in the projectile hit event, and it'll refer to the projectile itself